logging_config.py

```diff
@@ -0,0 +1,19 @@
+import logging
+
+
+def setup_logging() -> logging.Logger:
+    logger = logging.getLogger(__name__)
+    logger.setLevel("DEBUG")
+    formatter = logging.Formatter("{asctime} - {levelname} - {message}", style="{", datefmt="%Y-%m-%d %H:%M")
+
+    console_handler = logging.StreamHandler()
+    console_handler.setLevel("DEBUG")
+    console_handler.setFormatter(formatter)
+    logger.addHandler(console_handler)
+
+    file_handler = logging.FileHandler("app.log", mode="a", encoding="utf-8")
```

Comment:
Per task specification, we have only console log requirements
![image](https://github.com/user-attachments/assets/cd328083-e4ca-4e09-a280-9af34e39f1bc)